backend_new/src/controllers/application.controller.ts

```diff
@@ -77,8 +85,15 @@ export class ApplicationController {
     operationId: 'create',
   })
   @ApiOkResponse({ type: Application })
-  async create(@Body() dto: ApplicationCreate): Promise<Application> {
-    return await this.applicationService.create(dto, false);
+  async create(
+    @Body() dto: ApplicationCreate,
+    @Request() req: ExpressRequest,
+  ): Promise<Application> {
+    return await this.applicationService.create(
+      dto,
+      false,
+      mapTo(User, req.user),
```

Comment:
Noticing we didn't take in the req here in the old backend, could you remind me why we're making that change?

Reply:
ah, yeah so in the old backend we were scoping a lot of our services via the request. So we both had access to the request in most of the services, but it also meant that the services was being reinstantiated per request.

So as part of this we dropped the request scoping, and have to pass request around, but it also means we only instantiate the services once now

Reply:
Cool! Thank you for the explanation